### ISSUE TITLE
[FEATURE] "Add more fields" button need to be aligned within the Environment Variables section

### ISSUE THREAD

**Issue by minump** [rokwire/rokwire-building-blocks-api#975]

**Comment by minump** [comment]
<img width="1233" alt="Screen Shot 2022-07-28 at 10 59 53 AM" src="https://user-images.githubusercontent.com/22112888/181584214-bb712a2f-2f37-40d0-8dd3-57e8f40539bc.png">
Tried varies position arguments and css position styles. Not able to move the button position. Would renaming the button like this work?

**Comment by sandeep-ps** [comment]
Yes, that sounds like a reasonable workaround for now. Please proceed with this.

**Comment by bingzhang** [comment]
The alignment looks good to me. But it seems when I click the button of Add environment variables, there is overlapping. Please refer to the screenshot.

<img width="1253" alt="Screen Shot 2022-09-12 at 1 46 40 PM" src="https://user-images.githubusercontent.com/5349154/189732147-b5783840-2f02-4944-a3e7-4c28697435e9.png">
